theano/contrib/mkl/mkl_ndarray.c

```diff
@@ -1139,7 +1540,23 @@ static PyMethodDef MKLNdarray_methods[] = {
 
     {"zeros",
         (PyCFunction)MKLNdarray_Zeros, METH_STATIC | METH_VARARGS,
-        "Create a new MklNdarray with specified shape, filled with zeros."},
+        "Create a new MklNdarray with specified shape, filled ith zeros."},
+
+    {"__copy__",
+        (PyCFunction)MKLNdarray_View, METH_NOARGS,
```

Comment:
Before submit the new code, did you mean both `__copy__` and `__deepcopy__` should do the same deepcopy, and keep the current `copy` and `view` as it is?